src/app/directives/prev.directive.ts

```diff
@@ -0,0 +1,15 @@
+import { Directive, ElementRef, Input } from '@angular/core';
+
+@Directive({ selector: '[prev]' })
+export class PrevActivityDirective {
+    constructor(el: ElementRef) {
+        el.nativeElement.style.width = '75px';
+        el.nativeElement.style.lineHeight = '75px';
+        el.nativeElement.style.textAlign = 'center';
+        el.nativeElement.style.opacity = 1;
+        el.nativeElement.style.border = 'none';
+        el.nativeElement.style.color = 'white'
+        el.nativeElement.style.borderRadius = '50%';
+        el.nativeElement.style.background = '#D9534F';
```

Comment:
If I use the common file for both next and prev then I might have to use a class for the color of the buttons or pass the color values to the directive, won't that defeat the purpose of using directives.